ci/test_python.sh

```diff
@@ -58,13 +59,51 @@ timeout 60m pytest \
   tests -k "not ucxx"
 popd
 
-rapids-logger "Run local benchmark"
+rapids-logger "pytest explicit-comms (legacy dd)"
+pushd dask_cuda
+DASK_DATAFRAME__QUERY_PLANNING=False \
+DASK_CUDA_TEST_SINGLE_GPU=1 \
+DASK_CUDA_WAIT_WORKERS_MIN_TIMEOUT=20 \
+UCXPY_IFNAME=eth0 \
+UCX_WARN_UNUSED_ENV_VARS=n \
+UCX_MEMTYPE_CACHE=n \
+timeout 60m pytest \
```

Comment:
```suggestion
timeout 15m pytest \
```

Reply:
Since this is running only explicit comms we can substantially decrease the timeout. I think 15m may even be too high, but let's begin with that to be safe.

Additionally, are we switching to query-planning as default immediately or at least before 24.06 release? If not, I think we should keep on testing all with query-planning disabled, as that's really what users are gonna face.

Reply:
The latest version of dask uses query-planning on by default. So, unless we are using rapids-dask-dependency to override this default, we are indeed switching to query-planning as default immediately.

I'd be happy to test everything in dask-cuda with both `DASK_DATAFRAME__QUERY_PLANNING=True` and `DASK_DATAFRAME__QUERY_PLANNING=False` for `24.06`. It's possible that some downstream users of Dask  + RAPIDS will still be opting out for now. So, that's a fair point.

Reply:
If we're immediately switching then I'd say we can optimistically try not to test everything twice as the entire test set is a bit long, thus we can leave it as you wrote it and only decrease the timeout as per my suggestion above.

Reply:
Okay, makes sense - changed.